### ISSUE TITLE
Can start server after npm install

### ISSUE THREAD

**Issue by augnustin** [sahat/hackathon-starter#651]
```
npm start
npm info it worked if it ends with ok
npm info using npm@2.15.9
npm info using node@v4.5.0
npm info prestart hackathon-starter@4.3.0
npm info start hackathon-starter@4.3.0

> hackathon-starter@4.3.0 start /myapp
> node app.js

fs.js:808
  return binding.readdir(pathModule._makeLong(path));
                 ^

Error: ENOENT: no such file or directory, scandir '/myapp/node_modules/node-sass-middleware/node_modules/node-sass/vendor'
    at Error (native)
    at Object.fs.readdirSync (fs.js:808:18)
    at Object.getInstalledBinaries (/myapp/node_modules/node-sass-middleware/node_modules/node-sass/lib/extensions.js:122:13)
    at foundBinariesList (/myapp/node_modules/node-sass-middleware/node_modules/node-sass/lib/errors.js:20:15)
    at foundBinaries (/myapp/node_modules/node-sass-middleware/node_modules/node-sass/lib/errors.js:15:5)
    at Object.module.exports.missingBinary (/myapp/node_modules/node-sass-middleware/node_modules/node-sass/lib/errors.js:45:5)
    at module.exports (/myapp/node_modules/node-sass-middleware/node_modules/node-sass/lib/binding.js:15:30)
    at Object.<anonymous> (/myapp/node_modules/node-sass-middleware/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/myapp/node_modules/node-sass-middleware/middleware.js:3:12)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/myapp/app.js:20:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
    at startup (node.js:139:18)

npm info hackathon-starter@4.3.0 Failed to exec start script
npm ERR! Linux 4.4.0-21-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v4.5.0
npm ERR! npm  v2.15.9
npm ERR! code ELIFECYCLE
npm ERR! hackathon-starter@4.3.0 start: `node app.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the hackathon-starter@4.3.0 start script 'node app.js'.
npm ERR! This is most likely a problem with the hackathon-starter package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node app.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs hackathon-starter
npm ERR! Or if that isn't available, you can get their info via:
npm ERR! 
npm ERR!     npm owner ls hackathon-starter
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /myapp/npm-debug.log
```

Tried `npm rebuild node-sass` from [there](https://github.com/sass/node-sass/issues/1579) without success...

**Comment by haganbt** [comment]
@aug-riedinger I'm seeing exactly the same when deploying to heroku. Did you make any progress on resolving this?

**Comment by aug-riedinger** [comment]
Yes.

I think for this issue, the solution (though not very clean) was to run `npm install --unsafe-perm` instead of simply `npm install`.

Also I faced another issue of conflict between libraries which I solved the following way:

I edited my `package.json` to remove version constraints which were conflicting each others. I now can build a new environment without trouble with the following devDependencies:

```
    "eslint": "3.9.1",
    "eslint-config-airbnb-base": "11.0.0",
    "eslint-plugin-import": "2.2.0",
```

Good luck
